src/pages/Popup/App.tsx

```diff
@@ -21,14 +28,70 @@ function App() {
     ],
   };
 
+  function incrementNavigation(e: React.MouseEvent) {
+    const target = e.target as HTMLButtonElement;
+    updateNavigation(target.value as Pages);
+  }
+
   return (
     <div className="App">
       <header className="App-header">
-        <LogIn />
-        {/* <TaskHistory />
-        <TaskDisplay /> */}
-        {/* <TaskTracker /> */}
-        {/* <TaskEditor TaskData={dummyTask} /> */}
+        <p>current navigation: {current_navigation_state}</p>
+        <div className="nav-container">
+          <button
+            className="nav-button"
+            value={'Login'}
+            onClick={incrementNavigation}
```

Comment:
Why is it increment on each item?

Reply:
Every time a navigation button is clicked, we want to update the current state and also add that into history